packages/next/src/seo/route/schema.ts

```diff
@@ -0,0 +1,25 @@
+import { z } from 'zod';
+
+/**
+ * Schema for validating featured image data.
+ */
+const FeaturedImageSchema = z
```

Comment:
Putting aside why we need zod instead instead of inferring from either the component or the graphql type, this probably doesn't belong here top-level.